walkthroughs/eks/getting-started-with-appmesh/infrastructure/vpc_infrastructure.yaml

```diff
@@ -0,0 +1,263 @@
+Description:  This template deploys a VPC, with a pair of public and private subnets spread
+  across two Availability Zones. It deploys an internet gateway, with a default
+  route on the public subnets. It deploys a pair of NAT gateways (one in each AZ),
+  and default routes for them in the private subnets.
```

Comment:
Just a question: For folks with too many VPCs (*cough* me), is there a way to use an existing VPC?

Reply:
It can be done, but to make it simple we are providing a new VPC so the EKS cluster can reside in without breaking anything else into customers environment.